src/store/modules/util/actions.ts

```diff
@@ -26,6 +26,120 @@ const actions: ActionTree<UtilState, RootState> = {
     } catch (err) {
       console.error("error", err);
     }
+  },
+
+  // Get shopify enumeration description
+  async getShopifyEnumerations({ state, commit }, payload) {
+    const enumIds = payload.map((enumeration: any) => enumeration.productIdentifierEnumId)
+    const currentEnums = JSON.parse(JSON.stringify(state.enumerations));
+    const currentEnumIds = Object.keys(state.enumerations);
+
+    const enumIdFilter = enumIds.reduce((enums: any, enumId: any) => {
+      if(!currentEnumIds.includes(enumId)) {
```

Comment:
We could directly check if there is any value for key in currentEnums

Reply:
Sir, improved code.